config/vlan.py

```diff
@@ -76,6 +77,10 @@ def add_vlan(db, vid, multiple):
             if clicommon.check_if_vlanid_exist(db.cfgdb, vlan, "DHCP_RELAY"):
                 ctx.fail("DHCPv6 relay config for {} already exists".format(vlan))
 
+            # Enable STP on VLAN if PVST is enabled globally
```

Comment:
Do you think it is possible to put all these `if stp.is_global_stp_enabled(db.cfgdb)` into a single block or even a single function?  This will make it easier to maintain compared to having them scattered around.

Reply:
Moved the block to a function